backend/src/images/image.service.ts

```diff
@@ -0,0 +1,52 @@
+import { Injectable } from '@nestjs/common';
+import { InjectConnection, Knex } from 'nestjs-knex';
+import { IImage, Image } from '../tables';
+
+@Injectable()
+export class ImageService {
+  constructor(
+    @InjectConnection('group-d') private readonly knex_groupd: Knex,
+  ) {}
+
+  /**
+   * Change the path so that given the domain name and port of the backend server
+   * the frontend can find the right image
+   *
+   * @param image the image to process
+   *
+   * @author Kerbourc'h
+   */
+  formatImagePath(image: IImage) {
```

Comment:
so here the function takes an Image variable right?

Reply:
yes to replace image_path and make it so that the frontend only need to know the backend domain name and port